### PR TITLE
Add form received page

### PR DESCRIPTION
**Pull request by taniaprocha**

### CODE REVIEW

File: src/components/Feedback/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Link } from "gatsby";
+
+import Typography from "root/components/Typography";
+
+import "./index.css";
+
+const Feeback = () => {
```
Comment (filipebfernandes) [inline src/components/Feedback/index.js]: I would name this FormReceivedFeedback, otherwise, it's too general

**Comment by taniaprocha** [comment]
@filipebfernandes it's not finished yet, I'm reviewing with Ines. Sorry, I should create the PR as a draft.

**Comment by filipebfernandes** [comment]
ok @taniaprocha no problem. Ping me when it's done

**Comment by taniaprocha** [comment]
@filipebfernandes you can review now

**Pull request merged**